doc/maintaining/installing/index.rst

```diff
@@ -10,9 +10,9 @@ There are three ways to install CKAN:
 #. Install from Docker Compose
 
 From package is the quickest and easiest way to install CKAN, but it requires
-Ubuntu 16.04 64-bit 64-bit. **You should install CKAN from package if**:
+Ubuntu 14.04, 16.04 or 18.04 64-bit. **You should install CKAN from package if**:
```

Comment:
We dropped support for 14.04 on new CKAN versions in f65d992d19625b78fc6ebcde538acf49691ecfbe, as it only has [Extended Maintenance Support](https://ubuntu.com/about/release-cycle). We'll keep providing packages for CKAN versions that had 14.04 packages but not new ones, so let's not mention it here.

TL; DR mention only 16.04 and 18.04